pkg/controller/manager/manager_controller.go

```diff
@@ -381,6 +387,9 @@ func (r *ReconcileManager) processWebui(manager *v1alpha1.Manager, replicas *int
 			oldWebUI.ObjectMeta = v1.ObjectMeta{
 				Namespace: manager.Namespace,
 				Name:      *manager.Status.Webui.Name,
+				Labels: map[string]string{
```

Comment:
Why is this required?

Reply:
because manager adds label with its name to created resources which points to this manager as owner. So in case there would be other WebUI created by e.g. *cluster2* manager and we process this Delete without labels from *cluster1* manager then it'll delete *cluster2's* WebUI. Labels will protect resources created by other managers

Reply:
smart